lerobot-web-fe/src/components/Header.tsx

```
@@ -0,0 +1,22 @@
+interface AppbarProps {
+  isDrawerOpen: boolean;
+  robotStatus: boolean[];
+}
+
+export default function Appbar({ robotStatus }: AppbarProps) {
+  return (
+    <header className={styles.appbar}>
+      <h1 className={styles.title}>Lerobot Robot Arm</h1>
+      <div className={styles.robotIcons}>
+        {robotStatus.map((isActive) => (
+          <div className={`${styles.robotIcon} ${isActive ? styles.active : ''}`}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)